pathml/core/slide_data.py

```diff
@@ -251,6 +251,8 @@ def run(
         level=0,
         tile_pad=False,
         overwrite_existing_tiles=False,
+        write = False,
```

Comment:
Could condense these into one argument. If isinstance(write, str or pathlike), write to the path

Reply:
Agreed